app/ai/gemini.py

```
@@ -0,0 +1,88 @@
+    }
+    model_client = genai.GenerativeModel(model_name=model_name, generation_config=generation_config)
+    parts = [prompt]
+    if system:
+        parts = [f"SYSTEM:\n{system}\n\nUSER:\n{prompt}"]
+    try:
+        resp = model_client.generate_content(parts)
+        text = resp.text or "{}"
+        # Some SDK versions provide .candidates[0].content.parts; fall back to .text
+        import json
+        return json.loads(text)
+    except Exception as e:  # pragma: no cover
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)